maestro-android/build.gradle

```diff
@@ -80,10 +83,6 @@ tasks.register("copyMaestroAndroid", Copy) {
     into layout.buildDirectory.file(maestroAndroidApkDest)
 
     it.doLast {
-        if (JavaVersion.current() != JavaVersion.VERSION_1_8) {
```

Comment:
Why was this a requirement?

Reply:
Compiling Android modules with versions of Java higher than 8 is causing bytecode incompatibility issues on certain devices. We should preserve that behavior

Reply:
Which devices? Or do you mean a specific android api level?